packages/ddp-rate-limiter/ddp-rate-limiter.js

```diff
@@ -64,10 +64,11 @@ DDPRateLimiter.setErrorMessage = function (message) {
  * Default = 10.
  * @param {number} timeInterval time interval in milliseconds after which
  * rule's counters are reset. Default = 1000.
+ * @param {function} callback function to be called after a rule is executed.
  * @locus Server
  */
-DDPRateLimiter.addRule = function (matcher, numRequests, timeInterval) {
-  return rateLimiter.addRule(matcher, numRequests, timeInterval);
+DDPRateLimiter.addRule = function (matcher, numRequests, timeInterval, cb) {
```

Comment:
Nit: Any reason not to just call this `callback` instead of abbreviating it?  Would be nice to match the docs above it and use the same name like the other vars do.